nucleus/src/main/java/za/co/absa/subatomic/infrastructure/team/TeamAutomationHandler.java

```diff
@@ -64,21 +64,40 @@ public void createNewTeam(TeamEntity newTeamEntity) {
                 new TeamMemberId(newTeamEntity.getCreatedBy().getMemberId()),
                 teamSlackIdentity);
 
+        AtomistMemberBase atomistMemberBase = new AtomistMemberBase(
+                teamMemberEntity.getFirstName(),
+                teamMemberEntity.getDomainUsername(),
+                teamMemberSlackIdentity);
+
         TeamCreatedWithDetails newTeam = new TeamCreatedWithDetails(teamCreated,
-                new AtomistMemberBase(
-                        teamMemberEntity.getFirstName(),
-                        teamMemberEntity.getDomainUsername(),
-                        teamMemberSlackIdentity));
+                atomistMemberBase);
 
         ResponseEntity<String> response = restTemplate.postForEntity(
                 atomistConfigurationProperties.getTeamCreatedEventUrl(),
                 newTeam,
                 String.class);
 
         if (response.getStatusCode().is2xxSuccessful()) {
-            log.info("Atomist has ingested event successfully: {} -> {}",
+            log.info("Atomist has ingested teamCreatedEvent successfully: {} -> {}",
                     response.getHeaders(), response.getBody());
         }
+
+        // Raise event for teamSlackChannelCreated
+        teamSlackChannelCreated ingestableObject = new teamSlackChannelCreated(
+                teamCreated, atomistMemberBase);
+
+        ResponseEntity<String> responseteamSlackChannelCreated = restTemplate.postForEntity(
+                atomistConfigurationProperties
+                        .getMemberRemovedFromTeamEventUrl(),
+                ingestableObject,
+                String.class);
+
+        if (responseteamSlackChannelCreated.getStatusCode().is2xxSuccessful()) {
+            log.info("Atomist has ingested teamSlackChannelCreatedevent successfully: -> {}",
```

Comment:
Case fix to match atomist event naming case
```suggestion
            log.info("Atomist has ingested TeamSlackChannelCreatedEvent successfully: -> {}",
```